mycroft/configuration/mycroft.conf

```diff
@@ -111,14 +111,28 @@
     "sample_rate": 16000,
     "channels": 1,
     "record_wake_words": false,
-    "wake_word": "hey mycroft",
-    "phonemes": "HH EY . M AY K R AO F T",
+    "record_utterances": false,
```

Comment:
I think this is an unrelated addition

Reply:
yup, this one escaped me, should i still leave it? it is a parameter i added in another already merged PR but did not add to config file